hadoop-hdds/rocksdb-checkpoint-differ/src/main/java/org/apache/ozone/rocksdiff/RocksDBCheckpointDiffer.java

```diff
@@ -305,7 +314,7 @@ public void setCurrentCompactionLog(long latestSequenceNum) {
 
   @Override
   public void close() {
-    synchronized (compactionListenerWriteLock) {
+    synchronized (compactionDagLock) {
```

Comment:
If its a singleton class we can synchronize on "this". We do not need to create another object for synchronization.

Reply:
done.